packages/istanbul-reports/lib/none/index.js

```diff
@@ -5,6 +5,10 @@
  */
 const { ReportBase } = require('istanbul-lib-report');
 
-class NoneReport extends ReportBase {}
+class NoneReport extends ReportBase {
```

Comment:
Please undo this change as the ReportBase class has been fixed.

Reply:
done